README.rst

```diff
@@ -1,254 +1,22 @@
-.. image:: https://cloud.githubusercontent.com/assets/15197846/21435381/96e50466-c89f-11e6-8f0c-95267da389cf.png
+.. image:: https://cloud.githubusercontent.com/assets/5716520/24837754/982064c4-1d3b-11e7-9610-bbdd758c97b5.png
+    :target: https://coala.io/
 
-coala: Linting and fixing code for all languages
-------------------------------------------------
+Visit coala.io for More Information
+-----------------------------------
```

Comment:
because we *really* want people to go there and we can't put a link in a heading but it's too small outside so I'd rather do both :P